src/components/Inventory/AddInventory.js

```diff
@@ -0,0 +1,331 @@
+                    alt="Error Icon"
+                    className="form-error__icon"
+                  />
+                  This field is required
+                </p>
+              ) : null}
+              <br></br>
+
+              {/* Description Input */}
+              <label className="form-label">Description</label>
+              <textarea
+                name="description"
+                value={formData.description}
+                onChange={handleChange}
+                placeholder="Please enter a brief item description..."
+                className={`form-input--description ${
+                  errors.description ? "form-input--error" : ""
+                }`}
+              />
+              {errors.description ? (
+                <p className="form-error">
+                  <img
+                    src={ErrorIcon}
+                    alt="Error Icon"
+                    className="form-error__icon"
+                  />
+                  This field is required
+                </p>
+              ) : null}
+              <br></br>
+
+              {/* Category selection */}
+              <label className="form-label">Category</label>
+              <select
+                name="category"
+                value={formData.category}
+                onChange={handleChange}
+                className={`form-input ${
+                  errors.category ? "form-input--error" : ""
+                }`}
+              >
+                <option value="" disabled>
+                  Please select
+                </option>
+                <option value="Electronics">Electronics</option>
+                <option value="Gear">Gear</option>
+                <option value="Apparel">Apparel</option>
+                <option value="Accessories">Accessories</option>
+                <option value="Health">Health</option>
+              </select>
+              {errors.category ? (
+                <p className="form-error">
+                  <img
+                    src={ErrorIcon}
+                    alt="Error Icon"
+                    className="form-error__icon"
+                  />
+                  This field is required
+                </p>
+              ) : null}
+              <hr className="inventory-card__divider--hidden"></hr>
+            </section>
+            <section className="form--right">
+              <h2 className="form-heading">Item Availability</h2>
+
+              {/* status selection */}
+              <label className="form-label">Status</label>
+              <div className="form-input--status">
+                <input
+                  type="radio"
+                  value="In Stock"
+                  name="status"
+                  onChange={handleChange}
+                  checked={formData.status === "In Stock"}
+                />
+                In stock
+                <input
+                  type="radio"
+                  value="Out of Stock"
```

Comment:
An observation is that you are using these two strings: "In Stock" and "Out of Stock" in multiple places. Best would be to move them to new constants such as:

```
const IN_STOCK = "In Stock";
const OUT_OF_STOCK = "Out of Stock";

```
That way if any of these strings change in future you only have one place to change it while rest of the code will remain untouched